src/tribol/mesh/MfemData.hpp

```diff
@@ -0,0 +1,1390 @@
+// Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+// other Tribol Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (MIT)
+
+#ifndef SRC_MESH_MFEMDATA_HPP_
+#define SRC_MESH_MFEMDATA_HPP_
+
+#include <set>
+#include <vector>
+
+#include "mfem.hpp"
+
+#include "axom/core.hpp"
+#include "redecomp/redecomp.hpp"
+
+#include "tribol/types.hpp"
+#include "tribol/common/Parameters.hpp"
+#include "tribol/mesh/MethodCouplingData.hpp"
+
+namespace tribol
+{
+
+/**
+ * @brief Facilitates transfer to submesh low-order refined mesh
```

Comment:
"transfer 'to/from' submesh and related low order refined mesh"? Reword for clarity.

Reply:
Reworded